app/routes/register-your-interest/index.js

```diff
@@ -60,7 +60,7 @@ module.exports = [
     handler: async (request, h) => {
       await sendEmail(registerYourInterest, request.payload.emailAddress)
       await sendDefraIdRegisterYourInterestMessage(request.payload.emailAddress)
-      return h.redirect('registration-complete', { ruralPaymentsAgency })
+      return h.redirect('register-your-interest/registration-complete', { ruralPaymentsAgency })
```

Comment:
Should this be URL not be the following?

`defra-id/register-your-interest/registration-complete`

Reply:
Don't think so as it is a redirect so it will go to the get endpoint for that route, which redirects it to defra id screen if defra id is enabled
if it was an h.view it would have to point to the defra id screen specifically.